src/components/Dialog/Dialog.scss

```diff
@@ -9,7 +9,7 @@ $block: '.#{variables.$ns}dialog';
     position: relative;
     display: flex;
     flex-direction: column;
-    width: var(--_--width, var(--g-dialog-width));
```

Comment:
why here is double var()

its ok to use `var(--g-dialog-width), --_--width)`, isn't it?

@amje

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/var#syntax

Reply:
@saxumcordis No, @ValeraS is right, syntax is incorrect